src/main/kotlin/me/spradling/gift/core/database/aws/RDSGiftCommitStorageClient.kt

```diff
@@ -54,21 +55,44 @@ class RDSGiftCommitStorageClient @JsonCreator constructor(
     val accounts = mutableListOf<Account>()
     var resultSet = connection.createStatement().executeQuery(getQuery)
 
-    while(resultSet.next()) {
+    while (resultSet.next()) {
```

Comment:
nit: this is kind of duplicate logic from `getAccounts`, you could reuse that method and just return the first!

Reply:
I'll address this later in a follow up.